source/_docs/pantheon-workflow.md

```diff
@@ -12,13 +12,17 @@ Every Pantheon site comes with three environments: Dev, Test, and Live. Separate
   <p>Optimize your dev team and streamline internal workflows. Pantheon delivers custom workshops to help development teams master our platform and improve their internal DevOps.</p>
 </div>
 
-## Code Moves Up, Content Moves Down
-<img src="/source/docs/assets/images/workflow.png" alt="Dev Test and Live icon" style="border:0;margin-left:auto;margin-right:auto;display:block;">
+## Code VS Content
```

Comment:
Suggest reverting header changes - this doesn't summarize the section as effectively as the original h2